dwave/system/temperatures.py

```diff
@@ -218,7 +218,7 @@ def maximum_pseudolikelihood_temperature(bqm = None,
        >>> bqm = dimod.BinaryQuadraticModel.from_ising({}, {e : 1-2*random() for e in sampler.edgelist})
        >>> sampleset = sampler.sample(bqm, num_reads=100, auto_scale=False)
        >>> T,T_bootstrap =  maximum_pseudolikelihood_temperature(bqm,sampleset) 
-       >>> print('Effective temperature ',T)
+       >>> print('Effective temperature ',T)    # doctest: +SKIP
```

Comment:
```suggestion
       >>> print('Effective temperature ',T)    # doctest: +SKIP
       Effective temperature  0.24066488780293813
```

Example result running this on Advantage_system4.1